### PR TITLE
Initial code

### PR DESCRIPTION
**Pull request by jonhawkes**
Signed-off-by: Jon Hawkes <jon.hawkes@uk.ibm.com>

### CODE REVIEW

**Comment by kenfinnigan** [comment]
I think it's best if all the Liberty specific pieces are in their own profile.

In addition, it shouldn't have a parent POM that's Liberty specific.

File: src/main/java/application/rest/HealthEndpoint.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package application.rest;
```
Comment (Emily-Jiang) [inline src/main/java/application/rest/HealthEndpoint.java]: please add copyright for service-a and service-b.

**Review by Emily-Jiang — CHANGES_REQUESTED** [review]

File: src/main/java/application/rest/HealthEndpoint.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.health.Health;
+import org.eclipse.microprofile.health.HealthCheck;
+import org.eclipse.microprofile.health.HealthCheckResponse;
+
+@Health
+@WebListener
+@ApplicationScoped
+public class HealthEndpoint implements HealthCheck, ServletContextListener {
```
Comment (kenfinnigan) [inline src/main/java/application/rest/HealthEndpoint.java]: Not sure we should be using Servlet spec bits in MP, as it's technically not needed for MP. Not all implementations will be based on Servlet

**Comment by jonhawkes** [comment]
Fixes https://github.com/eclipse/microprofile-service-mesh-service-a/issues/3 but for service-b!

**Pull request closed without merging**